tools/pipelines/build-client.yml

```diff
@@ -97,19 +92,16 @@ pr:
     - release/*
   paths:
     include:
-    - .prettierignore
-    - packages
+    # Trigger on any edits to files in the root
+    - "*"
```

Comment:
Will this trigger just for files at the root, or for any file under the root (i.e. every file in every subfolder)?

Reply:
I can't find a clear answer in the docs. I _think_, since they support `**` as well, that `*` is single level, and `**` is everything under the path, but that's really just an assumption. @alexvy86 Do you happen to know?

Reply:
😆 commented before seeing your comment Tyler. So I'll continue the chain and see if maybe @Abe27342 happens to know.

Reply:
From the docs I would think the same thing Tyler said, but I'm also not 100% sure. Seems easy enough to test by making a change under server/ but would have to be after we merge this.

Reply:
I'd say go ahead and merge and double check I guess.

Flip side:  In that case, might your changes to remove the trailing `/*` on other paths imply a now-deeper search whereas currently they might be single layer?

Reply:
> Flip side: In that case, might your changes to remove the trailing /* on other paths imply a now-deeper search whereas currently they might be single layer?

The single layer was a mistake, I'm pretty sure. I can't think of a reason why those packages would only trigger when their root files (but not `src`, for example) are edited.